<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aesthetic Faces Dataset Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1223</width>
           <height>733</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableWidget" name="tbl_dataset">
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="rowCount">
             <number>1</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <row/>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Image</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Landmarks</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Age</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Aesthetic</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ethnicity</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Haircolor</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Skin type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Image_ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>hair</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>hair2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>HaircolorName</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGenerate_Dataset"/>
    <addaction name="actionLoad_Dataset"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Dataset"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGenerate_Dataset">
   <property name="text">
    <string>Generate Dataset</string>
   </property>
  </action>
  <action name="actionLoad_Dataset">
   <property name="text">
    <string>Load Dataset</string>
   </property>
  </action>
  <action name="actionSave_Dataset">
   <property name="text">
    <string>Save Dataset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
